app/controllers/api/v1/events_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::EventsController < ApplicationController
+  def index
+    render json: EventSerializer.new(Event.all)
```

Comment:
As we discussed, let's keep this as `Event.all` for now. Will discuss with FE team if we should just return future events.